src/Struct/FindologicProduct.php

```diff
@@ -1129,4 +1145,21 @@ protected function getCatUrlPrefix(): string
 
         return rtrim($path, '/');
     }
+
+    protected function filterThumbnails(MediaThumbnailCollection $thumbnails): MediaThumbnailCollection
```

Comment:
If your method name is generic like this, either split their contents up into multiple smaller methods or add a docblock describing what actually is filtered. Also a method called `filterThumbnails` shouldn't be responsible for changing their order as well.